src/common/imageio_rawspeed.cc

```diff
@@ -120,7 +120,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed(dt_image_t *img, const char *filena
 
 #ifdef __WIN32__
   const size_t len = strlen(filename) + 1;
-  wchar_t filen[len];
+  wchar_t filen[MAX_PATH];
   mbstowcs(filen, filename, len);
```

Comment:
> There are lot of other code pieces which are enclosed in some shape or form in conditional compiles like _WIN32 or __WIN32__or G_OS_WIN32. First I need to unify them using at least the same conditional, and then carefully review all as these code parts, as most likely they were not (fully) tested.

I think it should not be done in this pr, but after. Let's keep this as small as it can be.